solidity/ecdsa/deploy/01_deploy_sortition_pool.ts

```diff
@@ -13,7 +13,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   const SortitionPool = await deployments.deploy("SortitionPool", {
     from: deployer,
-    args: [TokenStaking.address, T.address, POOL_WEIGHT_DIVISOR],
+    args: [T.address, POOL_WEIGHT_DIVISOR],
```

Comment:
`TokenStaking` on line 11 can be removed now

Reply:
Good catch. Fixed in https://github.com/keep-network/keep-core/pull/2920/commits/f5da4a4ae51e314233bf7acf450fa15fcb5fac26.